src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs

```diff
@@ -831,4 +831,96 @@ private void debugPrint()
         }
         #endregion Private
     }
+
+    /// <summary>
+    /// A struct containing frame informations for a LottieAnimationView.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public readonly struct LottieFrameInfo
```

Comment:
can I ask why "struct" is used? impossible to use class?

Reply:
I used `readonly struct` to prevent changing properties by user.
When there is a method or property returning LottieFrameInfo type, such as,
```
class SomeClass
{
  public LottieFrameInfo FrameInfo { get; set; }
}
```
`readonly struct` will prevent user editing it like below,
```
var instance = new SomeClass();

// Build error
instance.FrameInfo.StartFrame = 3;
```